easybuild/easyconfigs/n/ncregrid/ncregrid-1.4b-goolf-1.4.10.eb

```diff
@@ -0,0 +1,47 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2014 The Cyprus Institute
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+# License:: MIT/GPL
+#
+##
+easyblock = 'ConfigureMake'
+
+name = 'ncregrid'
+version = '1.4b'
+
+homepage = 'http://www.pa.op.dlr.de/~PatrickJoeckel/ncregrid/'
+description = """ncregrid is a tool (written in FORTRAN-90) for data
+ transfer of gridded 2- and 3-dimensional (spatial) geophysical/geochemical
+ scalar fields between grids of different resolutions"""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchainopts = {'optarch': True, 'usempi': True, 'pic': True}
+
+source_urls = ['http://www.pa.op.dlr.de/~PatrickJoeckel/ncregrid/']
+sources = ['%(name)s.tar.Z']
```

Comment:
I'm running into this issue:

```
tar (child): compress: Cannot exec: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
```

because `compress` (used by `tar`) is not available on the system...

adding `compress` as a build dep is probably wise, but this also requires that https://github.com/hpcugent/easybuild-framework/issues/1376 gets fixed (which may be non-trivial to fix without introducing regressions elsewhere)

@wpoely86: are you up for taking a closer look at https://github.com/hpcugent/easybuild-framework/issues/1376?